### DISCUSSION TITLE
Ansible- Exception during task execution(Download and extract app content)

### DISCUSSION THREAD

**Discussion by Mohandkajuluri** [learndevopsonline/learndevopsonline#1588] [Q&A] [answered]
Hi, I'm unable to solve this, could you please help.

Workstation IP: 3.87.37.159

![Screenshot (25)](https://github.com/learndevopsonline/learndevopsonline/assets/144593251/d59e93b6-adc9-4ff3-b01f-fb6710f4d1f2)
![Screenshot (28)](https://github.com/learndevopsonline/learndevopsonline/assets/144593251/21919d88-72da-4314-baaa-cfe4229a0b8d)

**Comment by sreedharm07** [comment, marked as answer]
Hi @Mohandkajuluri 
add the remote_src line
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ba903353-2bf5-45dc-afe5-75be13802f1d)


![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/6a5d5bb1-54a1-4243-bd86-87ce1c143e5d)

**Comment by Mohandkajuluri** [comment]
Ok, will do that.
Thank you.
